packages/core/src/services/documentLogs/create.ts

```diff
@@ -4,33 +4,31 @@ import {
   Result,
   Transaction,
 } from '@latitude-data/core'
-import { Commit, DocumentLog, Workspace } from '$core/browser'
-
-import { assignDocumentLogToProviderLog } from '../providerLogs'
+import { Commit, DocumentLog } from '$core/browser'
 
 export type CreateDocumentLogProps = {
-  workspace: Workspace
-  uuid: string
-  documentUuid: string
   commit: Commit
-  resolvedContent: string
-  parameters: Record<string, unknown>
-  customIdentifier?: string
-  duration: number
-  providerLogUuids: string[]
+  data: {
+    uuid: string
+    documentUuid: string
+    parameters: Record<string, unknown>
+    resolvedContent: string
+    customIdentifier?: string
+    duration: number
+  }
 }
 
 export async function createDocumentLog(
   {
-    workspace,
-    uuid,
-    documentUuid,
+    data: {
+      uuid,
+      documentUuid,
+      resolvedContent,
+      parameters,
+      customIdentifier,
+      duration,
+    },
```

Comment:
Is nicer in services to have `data` and other parameters that are models like `commit`. This way is clear what's data for the new entity and what are other entities.